google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -489,14 +491,18 @@ def delete force: nil
         #
         # @!group Table
         #
-        def create_table table_id, name: nil, description: nil
+        def create_table table_id, name: nil, description: nil,
+                         encryption_configuration: nil
           ensure_service!
           new_tb = Google::Apis::BigqueryV2::Table.new(
             table_reference: Google::Apis::BigqueryV2::TableReference.new(
               project_id: project_id, dataset_id: dataset_id,
               table_id: table_id
             )
           )
+          unless encryption_configuration.nil?
```

Comment:
Any reason this isn't configured in the `tap` callback below?

Reply:
Done. Removed the `encryption_configuration` arg from `create_table`.